README.rst

```diff
@@ -51,6 +51,10 @@ The ``-I`` flag can be used for a list of certain words to allow that are in the
 
 The ``-L`` flag can be used to allow certain words that are comma-separated placed immediately after it.  **Important note:** The list passed to ``-L`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
 
+    codespell -x FILE, --exclude-file=FILE
+
+The ``-x`` flag can be used to specify a file FILE with lines to be excluded from being checked for errors or changed.  These lines in FILE should match the to-be-excluded lines exactly.
```

Comment:
I liked your "whole lines" from the earlier revision.

Reply:
I don't know what @DimitriPapadopoulos thinks, but whole might help for #2046 too.